Supress ClassMustBeFinal in Psalm results

fredden opened the pull request:
A new rule (ClassMustBeFinal) has been introduced in Psalm v6.6.0. These classes are being identified as candidates for being marked as final.

This should fix the test failures in https://github.com/maglnet/ComposerRequireChecker/pull/570

Ocramius reviewed (changes requested):
I think we can do this, but not in a **minor** release: it needs to be a major release. 

I'd say the rule in #570 needs to be suppressed in `psalm.xml`, for the time being.

fredden committed:
Supress ClassMustBeFinal in Psalm results

fredden commented:
@Ocramius that makes sense. I have updated this pull request to suppress the error instead and updated the target because the new configuration is only valid for with new Psalm.

Ocramius reviewed (approved):
Thanks @fredden!
